cli/lib/cube.ts

```diff
@@ -41,7 +43,11 @@ export function injectRevision(this: Pick<Context, 'variables' | 'log'>) {
     return term
   }
 
-  return map(({ subject, predicate, object, graph }: Quad) => {
+  function isDimension(predicate: Term) {
+    return dataset.pointer.any().has(schema.about, predicate).terms.length
+  }
```

Comment:
This way only predicates which are dimensions are considered for `schema:sameAs`

@l00mi do we want any additional constraint here?